upup/pkg/fi/cloudup/populate_instancegroup_spec.go

```diff
@@ -43,7 +43,7 @@ const (
 	defaultMasterMachineTypeDO  = "s-2vcpu-4gb"
 	defaultMasterMachineTypeALI = "ecs.n2.medium"
 
-	defaultDONodeImage  = "debian-9-x64"
+	defaultDONodeImage  = "debian-10-x64"
```

Comment:
One of the big moves in Kops 1.18 was switching to Ubuntu 20.04 as default image for the foreseeable future. Would it be a good idea to do the same for DO?
```suggestion
	defaultDONodeImage  = "ubuntu-20-04-x64"
```

Reply:
@timoreimann - Is it okay to switch to ubuntu 20.04 as default image for DO?

Reply:
Yeah totally fine with me, thanks for asking.

Reply:
Thanks @timoreimann. @srikiz could you change the PR and cherrypick to 1.19 branch? There is a beta release for 1.19 coming soon.